exercises/practice/space-age/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# Get the exercise name from the current directory
```

Comment:
These nested `CMakeLists.txt` files are all the same, right?

Do you think it'd be possible to convert these to symbolic links to a "master" copy of this file, or would that possibly break something?

Reply:
I REALLY want these exercises to work on windows so symlinks is a bad idea. Also, each exercise should be consistent for itself - we only use the top-level CMakeLists.txt for CI-testing.

Reply:
Yeah that's fair, I don't want to break Windows users.